README.md

```diff
@@ -1,7 +1,52 @@
+- `Collection` conformances for the `Array.DifferentiableView` type to make it easier to work with directly.
+- Some missing derivatives of standard library Swift functions like, `min(_:_:)`, `max(_:_:)`, `abs(_:)`, `atan2(_:_:)`, `Sequence.min()` and `Sequence.max()`
+- Provides a `differentiableMap(_:)` function for the Optional type. 
+
+## Differentiable Swift
+The goal of the Differentiable Swift language feature is to provide first-class, language-integrated support for differentiable programming, making Swift the first general-purpose, statically typed programming language to have automatic differentiation built in. Originally developed as part of the [Swift for TensorFlow](https://github.com/tensorflow/swift) project, teams at [PassiveLogic](https://passivelogic.com) and elsewhere are currently working on it. Differentiable Swift is purely a language feature and isn't tied to any specific machine learning framework or platform.
+
 ## Getting Started
+Differentiable Swift is present as an experimental language feature in Swift toolchains. Due to the incompleteness of its implementation, for best results we recommend using the newest release Swift toolchain from [swift.org](https://www.swift.org/download/). Or if you want to be on the bleeding edge to download one of the newest nightly toolchains. These tend to be more unstable however in terms of crashing. 
+
+In order to enable Differentiable Swift no special compiler flags are needed, but you do need to place the following:
+```swift
+import _Differentiation
+```
+in any file where differentiation will be used. The compiler will warn you about this if you do forget to add the above and try to use any differentiable Swift capabilities.
+
+When using this package and it's methods you only need to add the following:
+```swift 
+import Differentiation
+```
+Since the package also exports the `_Differentiation` module. 
+Make sure to add the package to you dependencies however:
+```swift
+dependencies: [
+  .package(url: "https://github.com/differentiable-swift/swift-differentiation", from: "0.0.1")
+]
+```
+And then adding the product to any target that needs access to the library:
+```swift
+.product(name: "Differentiation", package: "swift-differentiation"),
+```
 
 ## Contributing
+If you're missing functionality or something is broken please file an issue! If you find something Differentiable Swift related that's not working but isn't directly related to this library please report it in an issue in our [swift-differentiation-testing](https://github.com/differentiable-swift/swift-differentiation-testing) repo. Here we try to provide an easy way for people to report and track incorrect behaviour of the Differentiable Swift language feature. If you have code that reproduces your issue please add it to the issue so that we can more easily debug or help with the problem you're running into. 
```

Comment:
nothing to do for this PR but we should make an issue template in the testing repo with a section for a reproducer